templates/documentation/delivery-analytics/using-custom-domains.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: "Using custom domains"
-slug: "using-custom-domains"
+title: Using custom domains
+slug: using-custom-domains
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅